onelogin_aws_cli/userquery.py

```diff
@@ -6,19 +6,24 @@
 RolePrincipalPair = Tuple
 
 
-def user_choice(question: str, options: List[str]) -> str:
+def user_choice(question: str, options: List, renderer=lambda x: x):
```

Comment:
I've been a bit lax on the type hinting, but having a function would be a helpful place to have them:

`..., options: List[Any], renderer: Callable[[Any],str]=lambda x: x`

Reply:
Will do!